packages/salesforcedx-vscode-core/test/vscode-integration/commands/baseDeployRetrieve.test.ts

```diff
@@ -226,7 +226,7 @@ describe('Base Deploy Retrieve Commands', () => {
       await executor.run({ data: {}, type: 'CONTINUE' });
       const components = executor.lifecycle.doOperationStub.firstCall.args[0];
 
-      expect(getUserConfiguredApiVersionStub.calledOnce).to.equal(true);
+      expect(getUserConfiguredApiVersionStub.called).to.equal(true);
```

Comment:
Any specific reason we don't wanna call out the number of times it's getting called?

Reply:
With the changes I made, it's now called twice (when the unigration test runs end-to-end).  I could change it to:
```
expect(getUserConfiguredApiVersionStub.calledTwice).to.equal(true);
```
...but I was thinking it might be a bit brittle.  What are your thoughts?
...check for `getUserConfiguredApiVersionStub.called`?
...or check for `getUserConfiguredApiVersionStub.calledTwice`?